### DISCUSSION TITLE
First release, now do some testing!

### DISCUSSION THREAD

**Discussion by TheMMC** [TheMMC/In-Place_Upgrade_Helper#14] [Announcements]
I can't find any more bugs and every feature I personally want is working.
Time for a pre-release.

Comments and suggestions are welcome

German:
Ich finde keine Bugs mehr und alle Funktionen, die ich selbst haben will, sind jetzt drin.
Zeit für die erste Pre-Release.

Kommentare und Vorschläge werden gerne entgegen genommen

<hr /><em>This discussion was created from the release <a href='https://github.com/TheMMC/In-Place_Upgrade_Helper/releases/tag/v0.92'>First release, now do some testing!</a>.</em>

**Comment by QuiXMaDe** [comment]
Hello.
Items 21 and 23 in the menu are similar (setvarltscn2024 both).

**Reply by TheMMC** [reply]
Thanks for reporting! I just fixed it :)

**Comment by gibmol** [comment]
Beim Inplace Upgrade von Win 10 LTSC zu Win 11 LTSC krieg ich die Meldung "Der Product Key konnte nicht überprüft werden".
Woran könnte das liegen?
